channels.yaml

```
@@ -133,8 +133,18 @@ releases:
       etcd-arg:
         type: array
     agentArgs: *agentArgs-v1
+  - version: v1.21.8+k3s1
+    minChannelServerVersion: v2.6.0-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v2
```

This should be a `*` not `&` 
```suggestion
    serverArgs: *serverArgs-v2
```